canonical-web-and-design/multipass.run
# Add the missing prism and fix docs links

## Done
- Added prism to the project
- Made the docs link checker script only look for 404 errors

## QA
- Check the changes will resolve [the reported docs link errors](https://github.com/canonical-web-and-design/multipass.run/runs/4395327558?check_suite_focus=true)
- Open the docs in the demo and check there is no 404 loading the prism file.
